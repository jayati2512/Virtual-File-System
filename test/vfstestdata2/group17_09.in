createvfs demo7 
createvfs demo7 20480
mountvfs demo8
addfile / a /user/include/input.txt
listdir /a 1 /tmp/b
mountvfs demo7
listdir /a /tmp/b
listdir /a 3 /tmp/b
listdir /e 1 /tmp/b
makedir / c/d
listdir /c 0 /tmp/b
addfile /c file1 /user/include/input.txt
makedir / e
makedir /c
movedir /c
movedir /f /c
movedir /c /f
movedir /a /c
movedir /c /d
movedir /c /a
movedir /c /e
movedir /c /e
listdir /c 0 /tmp/g
listdir /e 0 /tmp/h
unmountvfs demo7
