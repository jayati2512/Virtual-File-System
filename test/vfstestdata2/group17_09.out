createvfs_FAILURE  VFS_INSUFFICIENT_ARGUMENTS
createvfs_FAILURE INVALID_SIZE
mountvfs_FAILURE DATA_FILE_NOT_FOUND
addfile_FAILURE 	VFS_NOT_MOUNTED
listdir_FAILURE 	VFS_NOT_MOUNTED
mountvfs_FAILURE DATA_FILE_NOT_FOUND	
listdir_FAILURE 	VFS_NOT_MOUNTED
listdir_FAILURE 	VFS_NOT_MOUNTED
listdir_FAILURE 	VFS_NOT_MOUNTED
makedir_FAILURE 	VFS_NOT_MOUNTED
listdir_FAILURE 	VFS_NOT_MOUNTED				
addfile_FAILURE 	VFS_NOT_MOUNTED
makedir_FAILURE 	VFS_NOT_MOUNTED
makedir_FAILURE 	VFS_NOT_MOUNTED
movedir_FAILURE 	VFS_NOT_MOUNTED
movedir_FAILURE 	VFS_NOT_MOUNTED
movedir_FAILURE 	VFS_NOT_MOUNTED
movedir_FAILURE 	VFS_NOT_MOUNTED
movedir_FAILURE 	VFS_NOT_MOUNTED
movedir_FAILURE 	VFS_NOT_MOUNTED
movedir_FAILURE 	VFS_NOT_MOUNTED
movedir_FAILURE 	VFS_NOT_MOUNTED
listdir_FAILURE 	VFS_NOT_MOUNTED
listdir_FAILURE 	VFS_NOT_MOUNTED
unmountvfs_FAILURE VFS_NOT_MOUNTED
